src/tensors/fibers.jl

```diff
@@ -326,7 +326,11 @@ end
 Base.summary(fbr::Tensor) = "$(join(size(fbr), "×")) Tensor($(summary(fbr.lvl)))"
 Base.summary(fbr::SubFiber) = "$(join(size(fbr), "×")) SubFiber($(summary(fbr.lvl)))"
 
-Base.similar(fbr::AbstractFiber) = Tensor(similar_level(fbr.lvl))
-Base.similar(fbr::AbstractFiber, dims::Tuple) = Tensor(similar_level(fbr.lvl, dims...))
+Base.similar(fbr::AbstractFiber) = similar(fbr, default(fbr), eltype(fbr), size(fbr))
+Base.similar(fbr::AbstractFiber, eltype::Type) = similar(fbr, default(fbr), eltype, size(fbr))
```

Comment:
either we should attempt to `convert(eltype, default(fbr))` here or we should use `zero(eltype)`.

Reply:
I think `convert(eltype, default(fbr))` is a better option. Done!